lib/action_policy/rspec/dsl.rb

```diff
@@ -68,7 +68,7 @@ def formatted_policy(policy)
 
   ::RSpec.shared_examples_for "action_policy:policy_rule_example" do |success, the_caller|
     if success
-      specify do
+      specify "is allowed" do
```

Comment:
What about

```suggestion
      specify "is authorized" do
```

Reply:
I think, "is allowed" makes more sense here, 'cause we checking for the rule application result, not the authorization fact. And in the end we would have smth like:

```
index?
    when authenticated
      is allowed
```

Which sounds right to me.